param/_utils.py

```diff
@@ -552,3 +588,18 @@ def _in_ipython():
         return True
     except NameError:
         return False
+
+_RUNNING_TASKS = set()
```

Comment:
As `_RUNNING_TASKS` is not a constant, shouldn't it be lowercase?

Reply:
Fair point, I do this relatively frequently but you're right I should change that and use lower case for mutable globals.